runtime/compiler/env/j9method.cpp

```diff
@@ -2387,15 +2393,48 @@ TR_ResolvedJ9Method::TR_ResolvedJ9Method(TR_FrontEnd * fe, TR_ResolvedMethod * o
 
 void TR_ResolvedJ9Method::construct()
    {
-#define x(a, b, c) a, sizeof(b) - 1, b, (int16_t)strlen(c), c
-
    struct X
       {
       TR::RecognizedMethod _enum;
       char _nameLen;
       const char * _name;
       int16_t _sigLen;
       const char * _sig;
+
+      typedef int8_t javaVersion_t;
+
+      javaVersion_t _nativeMinVersion;
+      javaVersion_t _nativeMaxVersion;
+
+// AllPastJavaVer and AllFutureJavaVer are used as lower and upper bounds for
```

Comment:
I find it a bit odd, in terms of style, to define the macros inside the struct - moving them ahead of the struct or something seems more logical because they are not scoped in anyway.